src/payment/PaymentPage.test.jsx

```diff
@@ -88,22 +121,19 @@ describe('<PaymentPage />', () => {
       expect(tree.toJSON()).toMatchSnapshot();
     });
 
-    it('should render the basket in a different currency', () => {
-      Cookies.result = {
```

Comment:
Given the new default above, deleting this `Cookies.result =` feels right.

Reply:
Added back in.